main/menu.js

```diff
@@ -271,6 +290,14 @@ function getAppMenuTemplate () {
           type: 'separator'
         },
         {
+          label: 'Preferences',
+          accelerator: 'Cmd+,',
+          click: () => showPreferencesWindow()
```

Comment:
That's not always the case: see https://github.com/feross/webtorrent-app/pull/130